### ISSUE TITLE
Could not install via packages.json

### ISSUE THREAD

**Issue by poovarasanm** [tomitrescak/meteor-uploads#190]
I am trying to install your library via packages.json configuration, but it is failing to download the library from NPM Registry. Do you have any plans to copy your library to npm registry?

I think many would prefer this facility as if we require to manage all the packages via packages.json.

Error:
   While building package npm-container:
   error: couldn't install npm package tomi:upload-server@2.2.0: Command failed: npm http GET
   https://registry.npmjs.org/tomi%3Aupload-server%402.2.0
   npm http 404 https://registry.npmjs.org/tomi%3Aupload-server%402.2.0

**Comment by tomitrescak** [comment]
Yup, I will do that eventually ;) Meteor 1.3 is around the corner.
